app/controllers/spaces_controller.rb

```diff
@@ -0,0 +1,13 @@
+class SpacesController < ApplicationController
+  
+  def home
+  	if !logged_in?
+  		redirect_to '/login'
+  	else 
```

Comment:
you want to do this check for all controllers - because as it stands, for example, I could create a user without being logged in.

you can create a `before_filter` in `ApplicationController` that will run for all controller actions, in all controllers (since they all extend `ApplicationController`).  In the filter you can do the check for logged in, and redirect there if they are not.